### PR TITLE
improve docs and document ingestion

### PR DESCRIPTION
**Pull request by emileten**
Work on https://github.com/NASA-IMPACT/active-maap-sprint/issues/752

- add docs about ingestion : there are two ways (1) direct connection to the DB and use pypgstac (for one or a few files) or (2) large number of items (use our IaC pipeline code -- link it).
- improve existing docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,58 @@
...
+or for a collection
+
+```
+pypgstac load --table collections test_collection.json
+```
+
+This requires 
+
+1. that you are allowed to connect to the database. Because of the security requirements mentioned above, you must go through a tunnel using the bastion host. [See these docs for more details](https://developmentseed.org/eoapi-cdk/#bastionhost-)
+2. [the configuration](https://stac-utils.github.io/pgstac/pypgstac/) for the database connection is present in your environment.
```
Comment (jjfrench) [inline README.md:47]: I think we want to avoid direct connections to the database and bastion host if we can help it

File: README.md [hunk 2]
```diff
@@ -1,14 +1,58 @@
...
+2. [the configuration](https://stac-utils.github.io/pgstac/pypgstac/) for the database connection is present in your environment. 
+
+
+### Indirect ingestion through the ingestion pipeline deployment
+
+For larger scale ingestions (a lot of items, for example), in MAAP we defined an approach where we : 
+1. First develop a python module that builds a STAC item out of a single input. It can be conformant to the [stactools-packages](https://github.com/stactools-packages) framework, or be as simple as [this example.](https://github.com/MAAP-Project/tri_seasonal_s1_sar_composites).
+2. Define an AWS lambda function that runs this module, an AWS queue where the lambda takes tasks from, and a file listing the tasks that fills the queue. 
+
+[This set up can be found here, defined as code](https://github.com/MAAP-Project/stac-pipeline).
```
Comment (jjfrench) [inline README.md:56]: Should this be moved over to https://github.com/MAAP-Project/stactools-pipelines ?

File: README.md (hunk 1 above)
Comment (emileten) [reply README.md]: Agreed, but shouldn't it be documented nevertheless?

File: README.md (hunk 2 above)
Comment (emileten) [reply README.md]: Done @jjfrench

**Comment by emileten** [comment]
@jjfrench can we merge ?

**Review by jjfrench — APPROVED** [review]

**Pull request merged**